### DISCUSSION TITLE
Consent tutorial

### DISCUSSION THREAD

**Discussion by Momo90k** [Shin-NiL/Godot-Android-Admob-Plugin#227] [General]
Hi.
Please provide or point me to a tutorial on how to set up GDPR and CCPA consent.
Thank you.

**Comment by attilaoroszdev** [comment]
That would be great, yeah. Or even just *some* documentation about how to use the `request_consent_info_update()` thing works would be great. Like what does the signal receive, is it a boolean? Do we need to do something with it? If the user didn't agree to personalised ads, should we manually flip that switch? And so on. It's confusing enough as it is, a little info would help a lot.

**Reply by attilaoroszdev** [reply]
I made a little plugin that will help with decyphering consent statuses. It also has a short tutorial for using Shin-Nil's plugin to obtain consent.

https://github.com/attilaoroszdev/godoteuconsentparserplugin
